diff -Nur hp2xx-3.4.2.orig/sources/Makefile hp2xx-3.4.2/sources/Makefile
--- hp2xx-3.4.2.orig/sources/Makefile	Sun Apr  7 17:45:53 2002
+++ hp2xx-3.4.2/sources/Makefile	Sun Jul  7 16:57:03 2002
@@ -35,18 +35,18 @@
 
 # Installation sites (GNU conventions):
 #
-prefix   = /usr/local
+prefix   = /usr/X11R6
 bindir   = $(prefix)/bin
 includedir = $(prefix)/include
 mandir   = $(prefix)/man
 man1dir  = $(mandir)/man1
-infodir  = $(prefix)/info
+infodir  = /usr/share/info
 
 # CC and CFLAGS set for gcc, but any ANSI-C compiler should work.
 # For non-gcc compilers, simply set CFLAGS = -O (to start with)
 #
 CC       = gcc
-CFLAGS	= -O2 -m486 -I/usr/X11R6/include $(DEFINES) $(EX_DEFS) -Wall -pedantic -Wuninitialized -Wunused -Wreturn-type -Wmissing-prototypes -Wstrict-prototypes  
+CFLAGS	= $(OPT) -I/usr/X11R6/include $(DEFINES) $(EX_DEFS) -Wall -pedantic -Wuninitialized -Wunused -Wreturn-type -Wmissing-prototypes -Wstrict-prototypes  
 
 #CFLAGS   = +O1 -Aa -D_HPUX_SOURCE -DSYSV -I/usr/include/X11R5 -I/usr/include/X11R4 $(DEFINES)
 
@@ -67,7 +67,7 @@
 #
 DEFINES   = -DUNIX -DHAS_UNIX_X11 
 PREVIEWER = to_x11
-ALL_LIBS  = -L/usr/lib/X11R6 -lpng -ltiff -ljpeg -lz -lX11 -lm     
+ALL_LIBS  = -L/usr/lib/X11R6 -lpng -ltiff -ljpeg -lz -lX11 -lm -lpdf
 #
 # Generic UNIX, no previewer
 #
@@ -173,9 +173,9 @@
 EX_OBJ =
 EX_DEFS =
 # Include PDF support
-#EX_SRC += to_pdf.c
-#EX_OBJ += to_pdf.o
-#EX_DEFS += -DUSEPDF
+EX_SRC += to_pdf.c
+EX_OBJ += to_pdf.o
+EX_DEFS += -DUSEPDF
 #
 # Epson Esc/P2 support 
 #
@@ -198,9 +198,9 @@
 EX_DEFS += -DJPG
 #
 # Include ATARI extras:
-# EX_SRC	+= to_pic.c to_pac.c
-# EX_OBJ	+= to_pic.o to_pac.o
-# EX_DEFS	+= -DPIC_PAC
+EX_SRC	+= to_pic.c to_pac.c
+EX_OBJ	+= to_pic.o to_pac.o
+EX_DEFS	+= -DPIC_PAC
 #
 #############################################################################
 # No user-serviceable parts below!
@@ -230,6 +230,8 @@
 	$(CC) $(LFLAGS) $(OBJS) $(ALL_LIBS) -o $(PROGRAM)
 	$(BINDCMD)
 
+all: $(PROGRAM) hp2xx.info
+
 default:
 	@echo "This makefile needs manual configuration! Edit it now!
 	@echo "After configuration, you may use the following commands:
@@ -311,15 +313,14 @@
 install:	install-bin install-info install-man
 
 install-bin:	$(PROGRAM)
-		-$(STRIP) $(PROGRAM)
-		-$(CP) $(PROGRAM) $(bindir)
-		-$(CHMOD) 755 $(bindir)/$(PROGRAM)
+		install -d $(DESTDIR)$(bindir)
+		install -m755 $(PROGRAM) $(DESTDIR)$(bindir)
 
 install-info:	hp2xx.info
-		-$(CP) hp2xx.info $(infodir)
-		-$(CHMOD) 644 $(infodir)/hp2xx.info
+		install -d $(DESTDIR)$(infodir)
+		install -m644 hp2xx.info $(DESTDIR)$(infodir)
 
 install-man:	../doc/hp2xx.1
-		-$(CP) ../doc/hp2xx.1 $(man1dir)
-		-$(CHMOD) 644 $(man1dir)/hp2xx.1
+		install -d $(DESTDIR)$(man1dir)
+		install -m644 ../doc/hp2xx.1 $(DESTDIR)$(man1dir)
 
